wire-ios-data-model/Source/MLS/MLSService.swift

```diff
@@ -1596,7 +1596,7 @@ public final class MLSService: MLSServiceInterface {
     ) async throws {
         do {
             logger.info("creating subgroup with id (\(id.safeForLoggingDescription))")
-            try await createGroup(for: id, parentGroupID: parentID)
+            _ = try await createGroup(for: id, parentGroupID: parentID)
```

Comment:
I fix a waning, do we need to pass the ciphersuite somewhere @KaterinaWire ?

Reply:
I think no, because we already send the ciphersuite to CC: inside of the` createGroup(...)`
- we take the defaultCipherSuite from the feature 
- and pass it to `createConversation(...)` in CC: https://github.com/wireapp/wire-ios/blob/1a022e72e4fcffab64d1b5970a614340d84f23c7/wire-ios-data-model/Source/MLS/MLSService.swift#L491